### ISSUE TITLE
erights Topic 4:  Rights Number / RigNumber field

### ISSUE THREAD

**Issue by rondlg** [fieldmuseum/EMu-Documentation#471]
DISCUSSION POINTS:  This is currently not required to be unique or filled in, and is not currently used to form a credit line or summary string, rather than to indicate a “number assigned by the institution to id the right or copyright issued”

**Comment by rondlg** [comment]
DECISION(S): 
Follow Events numbering convention for “departmental/area/project prefix + #”
Parse summary strings to display most commonly needed rights/credit/attribution  -- e.g., “(c) Rights Owner (year)”

DATA CHANGES:
[Data Changes Doc](https://docs.google.com/spreadsheets/d/1_nqm40TYUqier0FEHnNTn5JQCAKwAJybEcz7Wa6yr2I) # 1
Pre new development 
Group records in need of cleanup, and assign appropriate rights numbers in the MV notes, with type = “Rights Number”
Post development

**Comment by rondlg** [comment]
@magpiedin is this still development that we want? No is an ok answer.

Also, for `RigType`=Permit can you confirm that [this](https://docs.google.com/document/d/1MVz66S919YQ7icyk5ut0yi7gWBoL9jkRVAThdUSJF6Y/edit#heading=h.98btcya2ds9) is still a equired summary string update?

**Comment by magpiedin** [comment]
Yup -- We still need the summary string update (it's not implemented in v5.1 & v9.0)

For the development question -- I think we're all set; it looks like this was just a data change (using a numbering convention similar to event #'s)

**Comment by rondlg** [comment]
noted in structural changes for rights module discussion
